src/serializers/serializeStaticFilters.ts

```diff
@@ -25,11 +26,22 @@ function shapeCombinedFilterForApi(combinedFilter: CombinedFilter): StaticFilter
     : { [combinedFilter.combinator]: shapedFilters };
 }
 
+export function isNumberRangeValue(data: unknown): data is NumberRangeValue {
```

Comment:
Can we move `isNumberRangeValue` next to the `NumberRangeValue` model? I don't expect it to change much, but it's really import that type guards accurately reflect the model, because if they differ, runtime errors can creep in.

Reply:
I originally thought we may not want functions in models folder, and keep it for interfaces/types stuff only. But I remember we did have some type guard functions in models folder for answers-react-components as well. I will move it